docs/docsite/versions.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  active:
+   - devel
```

Comment:
Why have a single-space indent? Most people argue about zero vs. two-space... I'm in the no-indent camp FWIW.
https://github.com/ansible/awx-plugins/blob/0d569b5/.yamllint#L6C3-L8C28